src/schema/artwork/index.ts

```diff
@@ -804,7 +804,18 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return { label: "Certificate of authenticity", details: null }
         },
       },
-      pricingContext: PricingContext,
+      widthCm: {
+        description:
+          "If you need to render artwork dimensions as a string, prefer the `Artwork#dimensions` field",
+        type: GraphQLFloat,
+        resolve: ({ width_cm }) => width_cm,
+      },
+      heightCm: {
+        description:
+          "If you need to render artwork dimensions as a string, prefer the `Artwork#dimensions` field",
+        type: GraphQLFloat,
+        resolve: ({ height_cm }) => height_cm,
+      },
```

Comment:
When stitching the `pricingContext` into the artwork type, we only have access to the data resolved by graphql on the Artwork type. We weren't exposing the numerical dimensions of an artwork before, only the `dimensions` string values.

It feels unfortunate to expose these extra fields just because we need them for stitching. We couldn't think of an alternative though.

Reply:
Ah, hmm, I would have expected the `source` parameter for the stitched resolver would get access to that data 🤔 What _does_ it contain then? Is there the opportunity to improve this in the library?

Reply:
It contains whatever you query for in the `fragment` field. Feels like there should be a way to allow these stitched fields to optionally access the resolver args of their parent type.

Reply:
I agree, it makes sense for augmented fields to receive the same data as sibling fields, rather than having to ‘pollute’ the schema. Sounds like something to contribute.